modin/core/storage_formats/pandas/parsers.py

```diff
@@ -780,7 +781,7 @@ def parse(files_for_parser, engine, **kwargs):
         storage_options = kwargs.get("storage_options", {})
         chunks = []
         # `single_worker_read` just passes in a string path
-        if isinstance(files_for_parser, str):
+        if isinstance(files_for_parser, (str, Path)):
```

Comment:
ninja-fix of something else hidden in `TEST-` PR? [tut-tut](https://ru.wiktionary.org/wiki/tut-tut)...

if that is only needed for tests, one can call the parser like `pd.read_csv(str(path_obj))`

Reply:
It's needed for parquet tests. Updated

Reply:
https://github.com/modin-project/modin/issues/6015